test/build/config-loading-esm-package/config-loading-esm-package.test.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+const {setupBuildTest} = require('../../test-utils');
+
+const cwd = path.join(__dirname, 'TEST_BUILD_OUT');
+
+// Skip tests on node@10.x (expected to fail)
+describe = process.version.startsWith('v10') ? describe.skip : describe;
```

Comment:
What happens on Node 10 for this? Do we throw an error to the user?

Reply:
Good question! It throws Node's default error message which is basically `Unexpected token "export"` and points to the file. Clear enough that I don't *think* we need to wrap it, but I'm happy to

Reply:
If it’s not too much trouble, if there’s already a block of logic that’s only for ESM (that `import()` statement?) It’d be nice to at least throw something like `ESM Snowpack configuration not supported on Node 10. Please use CommonJS instead`, e.g.